app/views/content_items/html_publication/_print_meta_data.html.erb

```diff
@@ -2,20 +2,20 @@
   <%= image_tag "open-government-licence-min.png", class: "print-meta-data-licence" %>
 </p>
 <p>
-  © Crown copyright <%= @content_item.copyright_year %>
+  <%= t("html_publication.print_meta_data.copyright", year: @content_item.copyright_year) %>
 </p>
 <p>
-  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
+  <%= t("html_publication.print_meta_data.licence") %>
 </p>
 <p>
-  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
+  <%= t("html_publication.print_meta_data.third_party") %>
 </p>
 <p>
-  This publication is available at <%= @content_item.full_path(request) %>
+  <%= sanitize(t("html_publication.print_meta_data.available_at", url: @content_item.full_path(request))) %>
 </p>
 
 <% unless @content_item.isbn.blank? %>
     <p>
-      ISBN: <%= @content_item.isbn %>
+      <%= t("html_publication.print_meta_data.isbn") %>: <%= @content_item.isbn %>
```

Comment:
Similar question here about the `:` as with `Visit:`.

Reply:
Thanks, I'll add that to the translation here too